coalib/output/ConsoleInteraction.py

```diff
@@ -23,6 +23,39 @@
     RESULT_SEVERITY, RESULT_SEVERITY_COLORS)
 from coalib.settings.Setting import Setting
 
+from pygments import highlight
+from pygments.util import ClassNotFound
+from pygments.formatters import TerminalFormatter
+from pygments.formatters import TerminalTrueColorFormatter
+from pygments.filters import VisibleWhitespaceFilter
+from pygments.lexers import TextLexer, get_lexer_for_filename
+from pygments.style import Style
+from pygments.token import Token
+
+
+class BackgroundSourceRange(Style):
+    '''
+    Sub class of ``Style``.
+
+    This class sets styling properties such as ``background color``,
+    ``text-color`` and ``bold`` on the text.
+    '''
+    styles = {
+        Token: 'bold bg:#BB4D3E #111'
+    }
+
+
+class BackgroundMessage(Style):
+    '''
+    Sub class of ``Style``.
+
+    This class sets styling properties such as ``background color``,
+    ``text-color`` and ``bold`` on the text.
```

Comment:
do you want me to change them or remove entirely? Also since you wanted to know what these classes do, they are for highlighting the background and using a different foreground color in the highlighted message or sourcerange.